Recipes/Along-slope-velocities.ipynb

```diff
@@ -23,6 +23,7 @@
    "outputs": [],
```

Comment:
<p>lower case and more verbose? e.g., <code>latitude_slice</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/458/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>